<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWidget</class>
 <widget class="QWidget" name="RoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>一区域</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FirstZH"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>一区域英文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FirstEN"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="checkFirstEN">
       <property name="text">
        <string>检查名称</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>二区域</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondZH"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>二区域英文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondEN"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>三区域</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="srdZH"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>三区域英文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="srdEN"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>房间ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="roomID">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>房间数字</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="roomNum">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="checkRoomName">
       <property name="text">
        <string>生成并检查名称</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>房间名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="roomZN">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>房间英文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="roomEN">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>房间颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="roomColor"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>明显出口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outLE">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>实际出口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nowLE"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>房间描述</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="roomMessage"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="npclist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>68</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
